Documentation/ReleaseNotes.md

```diff
@@ -1,14 +1,5 @@
 # Microsoft Mixed Reality Toolkit release notes
 
-- [Version 2.4.0](#version-240)
-- [Version 2.3.0](#version-230)
```

Comment:
Do we want to cache previous versions in their own release notes pages? https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7442#discussion_r386317498

Reply:
I would say 'no' since the documentation for each release remains published.

Reply:
it is the upgrade guide (ex: breaking change mitigations) that feels most valuable to persist and it has not been removed from updating.md.

Reply:
Works for me. Will work well if we get the page switching between versions working instead of defaulting back to the root.

Reply:
i am opening an issue for 2.4.0 that will become a normal part of release management to handle this